src/js/rebuses.js

```diff
@@ -866,6 +866,11 @@ const rebuses = [
     symbols: ['🎁', '+', '🃏'],
     words: ['gift', 'card']
   },
+  {
+    symbols: ['_/\_'],
+    words: [Salute],
```

Comment:
Thanks for the addition. To make the build passing, you need to make the words property an array of strings: `words: ['Salute'],`